src/mainwindow.cpp

```diff
@@ -106,7 +106,7 @@ void MainWindow::setEditor()
     editor->setSizePolicy(QSizePolicy::Policy::Expanding, QSizePolicy::Policy::Expanding);
     editor->setAcceptDrops(false);
 
-    ui->verticalLayout_8->addWidget(editor);
+    ui->verticalLayout8->addWidget(editor);
```

Comment:
```suggestion
    ui->verticalLayout8->addWidget(editor);
```
We can give it a meaningful name.